examples/interactive_ml/app/README.md

```diff
@@ -0,0 +1,42 @@
+# Interactive ML App
```

Comment:
is this meant to be part of this PR?

Reply:
Müsste weg sein, wenn ich den main merge

Reply:
https://github.com/geo-engine/geoengine-python/pull/212

Reply:
ok, so merge it? :sweat_smile: